dynawo/sources/Solvers/SolverIDA/test/jobs/solverTestAlpha.dyd

```diff
@@ -13,6 +13,6 @@
 -->
 <dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
   <dyn:modelicaModel id="solverTestAlpha">
-    <dyn:unitDynamicModel id="solverTestAlpha" name="DYNSolverTestAlpha" moFile="DYNSolverTest.mo"/>
+    <dyn:unitDynamicModel id="solverTestAlphaModel" name="DYNSolverTestAlpha" moFile="DYNSolverTest.mo"/>
```

Comment:
We started to use the following coding style for this kind of things:

- modelicaModel id starts with a capital letter
- unitDynamicModel id starts with a small letter (that way, no need to add the suffix Model)

Same comment for the other tests

Reply:
Solved